frontend/src/components/Gym/GymStatus.tsx

```diff
@@ -31,15 +31,17 @@ const Status = styled.div<{ isOpen: boolean }>`
 `
 
 interface Props {
-  isOpen: boolean
+  isOpen: boolean | null
```

Comment:
are there any use cases for null?

Reply:
the changes here was because the API calls takes some time, and if we default to false, the page will render false for a good 1-2 seconds before rendering the correct state. I thought it would be better to render nothing than to render wrong info, although I'm not sure if this is the best way to resolve it.

Reply:
ah okok but might want to keep the height of the container even though there's no text/info so the buttons and other info don't move around too much